docs/how-to-display-operator-information-in-the-marketplace-ui.md

```diff
@@ -0,0 +1,72 @@
+# How to Display Operator Information in the Marketplace UI
+
+An operator's csv must contain additional information to be displayed properly within the Marketplace UI. The Marketplace expects the following information::
+
+```yaml
+metadata:
+  annotations:
+    categories: A list of comma separated categories that your operator falls under.
+    createdAt: The date that the operator was created.
+    description: A short description of the operator. Only the first 135 characters will appear.
```

Comment:
Isn't this already part of the CSV?

Reply:
The description is part of the CSV but this is meant to be a short description that will be displayed on the tile. If the short description is not present, the UI will use the long description for the tile and truncate. I did this to give people the option in case they have a long description whose first 135 characters don't display the meaning of the operator well enough

Reply:
`spec.description` exists but @galletti94 had mentioned that the UI team is using the `metadata.annotations.description` field to display a description on an operator tile - perhaps he could share more.

Reply:
Then this should be explained more clearly here.

Reply:
I will update the document:
```
metadata:
  annotations:
    description: -|
      A short description of the operator that will be displayed on the marketplace tile.
      If this annotation is not present, the `spec.description` value will be shown instead.
      In either case, only the first 135 characters will appear.
```

Reply:
Addressed in the latest commit.